exe/sufia_export

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+require 'rails'
+
+# get the current running directory & load the environment from there
+require ::File.join(Dir.pwd, 'config/environment')
```

Comment:
@mjgiarlo The Rails.root is nil at that point.  After much digging I found Bundler.default_gemfile which I will change to using.